tsdb/mapper.go

```diff
@@ -443,6 +451,8 @@ func (lm *LocalMapper) rewriteSelectStatement(stmt *influxql.SelectStatement) (*
 
 // expandWildcards returns a new SelectStatement with wildcards in the fields
 // and/or GROUP BY expanded with actual field names.
+// Tags and Measurements are included in the select if a `SELECT *` is issued
+// There is no longer an explicit `GROUP BY *` when tags are used in the `SELECT` clause
```

Comment:
Agreed, I thought the same thing twice, so looks like the voice in my head agrees with you.